front/lib/resources/data_source_view_resource.ts

```diff
@@ -112,6 +113,10 @@ export class DataSourceViewResource extends ResourceWithVault<DataSourceViewMode
       vault,
       dataSource
     );
+
+    await dataSourceView.setEditedBy(auth);
```

Comment:
Why not creating the datasourceView directly with `editedByUserId` & `editedAt`? Cause here we double the query